src/components/index.js

```diff
@@ -0,0 +1,33 @@
+import AppHeader from './AppHeader.js';
```

Comment:
Just replace the `import` with `export` and remove the export section.

Reply:
Changed index.js file to only export, to do this I had to also change the components so that they didn't have default exports